### PR TITLE
HIVE-21869 Clean up Kafka storage handler examples

### PR DESCRIPTION
**Pull request by kristopherkane**

### CODE REVIEW

File: kafka-handler/README.md [hunk 1]
```diff
@@ -1,42 +1,71 @@
...
+ 
+List of supported serializers and deserializers:
 
-|Supported Serializer Deserializer|
+|Supported Serializers and Deserializers|
 |-----|
 |org.apache.hadoop.hive.serde2.JsonSerDe|
 |org.apache.hadoop.hive.serde2.OpenCSVSerde|
-|org.apache.hadoop.hive.serde2.avro.AvroSerDe|
+|org.apache.hadoop.hive.serde2.avro.AvroSerDe*|
```
Comment (b-slim) [inline kafka-handler/README.md]: not sure what is the `*` at the end

File: kafka-handler/README.md [hunk 2]
```diff
@@ -1,42 +1,71 @@
...
 |org.apache.hadoop.hive.serde2.lazybinary.LazyBinarySerDe|
 |org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe|
 
-#### Table definition 
-In addition to the user defined payload schema Kafka Storage Handler will append additional columns allowing user to query the Kafka metadata fields:
+`*` This is just Apache Avro and not Confluent's serialization with schema registry integration.
```
Comment (b-slim) [inline kafka-handler/README.md]: nit...not sure if this is really needed.

File: kafka-handler/README.md [hunk 3]
```diff
@@ -47,71 +76,139 @@ In addition to the user defined payload schema Kafka Storage Handler will append
 
 List the table properties and all the partition/offsets information for the topic. 
 ```sql
-Describe extended kafka_table;
+DESCRIBE EXTENDED 
+  kafka_table;
 ```
 
-Count the number of records with Kafka record timestamp within the last 10 minutes interval.
+Count the number of records with where the record timestamp is within the last 10 minutes of query execution time.
```
Comment (b-slim) [inline kafka-handler/README.md]: with where does not sound correct to me

**Comment by b-slim** [comment]
did quick pass and left minor comments. thanks for the contribution

**Comment by kristopherkane** [comment]
Slim, I addressed all comments.  

Does the project prefer a JIRA patch over a GH PR? I did both for this one but subsequent JIRA only have patches.

**Comment by b-slim** [comment]
Jira patch is a must because that is how we run tests.
But as you know we can not review a patch as is so that is why we need a GH PR.
Thus we need both, usually we start by GH PR to trigger the code review then when done we move to test the patch and make sure it passes all the tests.

**Comment by b-slim** [comment]
LGTM thanks very much for this nice rewrite.

**Comment by belugabehr** [comment]
Issue resolved through JIRA:

https://issues.apache.org/jira/browse/HIVE-21869

**Pull request closed without merging**